subprojects/soak/soak.gradle.kts

```diff
@@ -35,3 +35,9 @@ dependencies {
 
     integTestDistributionRuntimeOnly(project(":distributions-full"))
 }
+
+tasks.register("soakTest") {
+    description = "Run all soak tests defined in the :soak subproject"
+    group = gradlebuild.CI_GROUP
```

Comment:
I am afraid this won't work anymore once "buildSrc" has the restrictions of a normal included build. Then the lifecycle project won't be on the classpath of this project. 

The group is only for documentation. It has no influence on the workings of the build.

If we look at this task as something in parallel to "embeddedIntegrationTest", I think we can just **not set** the group at all as these other test tasks also don't set a group.